pkgs/tools/security/fprintd/default.nix

```diff
@@ -47,13 +45,28 @@ stdenv.mkDerivation rec {
     libfprint
   ];
 
+  preConfigure = "./autogen.sh";
+
+  postPatch = ''
+    substituteInPlace tests/fprintd.py \
```

Comment:
Maybe add test python to `checkInputs`

Reply:
It did not work (test fails like there was no shebang substitutes), am I missing something ?